roles/scenario8/tasks/break.yml

```diff
@@ -1,8 +1,25 @@
 ---
 - name: execute tasks on compute nodes
-  include_tasks: break_computes.yml
-  when: inventory_hostname in groups['Compute']
+  include_tasks:
+    file: break_computes.yml
+    apply:
+      tags: break
+  when: (installer == 'tripleo' and inventory_hostname in groups['Compute']) or installer == 'podified'
```

Comment:
these Compute hardcoded, probably should be replaced with `compute_group_name` now?